src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -207,15 +207,15 @@ public void setType(EntryType type) {
     }
 
     /**
-     * Returns the contents of the given field, or null if it is not set.
+     * Use {@link #getFieldOptional} instead
      */
-    @Deprecated //Use getFieldOptional instead
+    @Deprecated
     public String getField(String name) {
         return fields.get(toLowerCase(name));
     }
 
     /**
-     * Returns the contents of the given field as an Optional.
+     * Returns the contents of the given field as an Optional. The optional may contain null
```

Comment:
No, it may not. The Optional will be empty in case it was supposed to contain null.